library/src/main/res/drawable/validate_button.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please, add prefix 'selector' at start so we would have selector_validate_button